src/renderer/chunk_meshing.zig

```diff
@@ -1188,42 +1186,43 @@ pub const ChunkMesh = struct { // MARK: ChunkMesh
 		}
 		self.mutex.unlock();
 
-		if(oldBlock.entityDataClass()) |class| {
-			class.onBreakClient(.{_x, _y, _z}, self.chunk);
-		}
-
 		var neighborBlocks: [6]Block = undefined;
 		@memset(&neighborBlocks, .{.typ = 0, .data = 0});
+
 		for(chunk.Neighbor.iterable) |neighbor| {
 			const nx = x + neighbor.relX();
 			const ny = y + neighbor.relY();
 			const nz = z + neighbor.relZ();
-			if(nx & chunk.chunkMask != nx or ny & chunk.chunkMask != ny or nz & chunk.chunkMask != nz) {
+
+			if(!self.chunk.liesInChunk(nx, ny, nz)) {
```

Comment:
The function actually works on a different coordinate space (world space relative coordinates), while you give it chunk space relative coordinates.
This is not relevant now, since the function only operates on voxelSIze=1 chunks, but it might be relevant in the future.

Reply:
Well, what do I do about it?

Reply:
You revert to the old logic, or add a function with the old logic with a name that clearly distinguishes it from `liesInChunk`.

Reply:
> clearly distinguishes it from `liesInChunk`.

Yeah, unlikely to happen, lets just get this over with.

```suggestion
			if(nx & chunk.chunkMask != nx or ny & chunk.chunkMask != ny or nz & chunk.chunkMask != nz) {
```